src/commands/mod.rs

```diff
@@ -44,6 +45,8 @@ pub enum Command {
     /// can submit proposals (such as a Motion Proposal) to be voted on by other neuron
     /// holders.
     MakeProposal(make_proposal::MakeProposalOpts),
+    /// Signs a request for a refund of ICP from the swap canister.
+    Refund(refund::RefundOpts),
```

Comment:
Nit: Refund could probably be applied to ledger transfers and neuron staking. Perhaps a more detailed name could avoid some user confusion. I recommend `RefundSwap` but open to counter suggestions.